src/main/java/io/github/seggan/slimypowersuits/machines/ElementForge.java

```diff
@@ -0,0 +1,153 @@
+package io.github.seggan.slimypowersuits.machines;
+
+import io.github.seggan.slimypowersuits.SlimyPowerSuits;
+import io.github.seggan.slimypowersuits.Utils;
+import io.github.seggan.slimypowersuits.lists.MiscItems;
+import io.github.thebusybiscuit.slimefun4.core.multiblocks.MultiBlockMachine;
+import io.github.thebusybiscuit.slimefun4.implementation.SlimefunPlugin;
+import io.github.thebusybiscuit.slimefun4.libraries.paperlib.PaperLib;
+import io.github.thebusybiscuit.slimefun4.utils.SlimefunUtils;
+import me.mrCookieSlime.CSCoreLibPlugin.cscorelib2.item.CustomItem;
+import me.mrCookieSlime.Slimefun.Lists.RecipeType;
+import me.mrCookieSlime.Slimefun.Objects.Category;
+import me.mrCookieSlime.Slimefun.api.Slimefun;
+import me.mrCookieSlime.Slimefun.api.SlimefunItemStack;
+import me.mrCookieSlime.Slimefun.cscorelib2.inventory.ItemUtils;
+import org.bukkit.Material;
+import org.bukkit.NamespacedKey;
+import org.bukkit.Sound;
+import org.bukkit.block.Block;
+import org.bukkit.block.BlockFace;
+import org.bukkit.block.BlockState;
+import org.bukkit.block.Dispenser;
+import org.bukkit.block.data.BlockData;
+import org.bukkit.block.data.type.Piston;
+import org.bukkit.block.data.type.PistonHead;
+import org.bukkit.entity.Player;
+import org.bukkit.inventory.Inventory;
+import org.bukkit.inventory.ItemStack;
+
+import java.util.List;
+
+/**
+ *
+ * This is a multiblock machine for crafting
+ * Slimy Power Suits materials.
+ * Heavily influenced by the Enhanced Crafting Table
+ * and Magical Workbench
+ *
+ * @author NCBPFluffyBear
+ */
+public class ElementForge extends MultiBlockMachine {
+
+    public static final RecipeType ELEMENT_FORGE_RECIPE = new RecipeType(
+        new NamespacedKey(SlimyPowerSuits.getInstance(), "element_forge_recipe"),
+        MiscItems.ELEMENT_FORGE,
+        "&cElement Forge",
+        "&7Used to create new elements"
+    );
```

Comment:
Make a getter for this...

Reply:
![image](https://user-images.githubusercontent.com/31554056/95668143-750eab80-0b35-11eb-84bc-26e952e9b1fe.png)
not sure how this is any better than the variable

Reply:
no make the variable `private static final` and make a method `getRecipeType()`

Reply:
getRecipeType() is a preexisting method, is this intentional

Reply:
at least do 

>
> 
> make the variable `private static final` and make a method returning the variable